packages/divider-block/src/settings.ts

```diff
@@ -163,8 +151,8 @@ export const settings: BlockSettings = {
                         {
                             id: HEIGHT_CUSTOM_ID,
```

Comment:
here as well. the `sectionHeading` with label `Block` is imo not necessary. Otherwise I would remove the `Block` within the label `Block Height` (line 144).